app/src/main/java/com/example/internetshop/presentation/activity/MainActivity.kt

```diff
@@ -93,13 +103,22 @@ class MainActivity : AppCompatActivity(), ContainerHolder {
         })
     }
 
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        return when (item.itemId) {
-            R.id.home -> {
-                supportFragmentManager.popBackStack()
-                true
-            }
-            else -> super.onOptionsItemSelected(item)
+    private fun setupAppBar() {
+        WindowCompat.setDecorFitsSystemWindows(window, false)
+        setSupportActionBar(binding?.toolbar)
+        supportActionBar?.setHomeAsUpIndicator(
+            (ResourcesCompat.getDrawable(resources, R.drawable.ic_back_arrow, null))
+        )
+    }
+
+    private fun updateLayoutParams() {
+        ViewCompat.setOnApplyWindowInsetsListener(binding?.appBar!!) { view, windowInsets ->
+            val insets = windowInsets.getInsets(WindowInsetsCompat.Type.systemGestures())
```

Comment:
Are you sure that you need to use inset type `systemGestures`? As I understand we need to use `systemBars()` here. Please double check

Reply:
fixed